tools/cli/adminTaskListCommands.go

```diff
@@ -87,8 +87,31 @@ func AdminListTaskList(c *cli.Context) {
 	}
 
 	fmt.Println("Task Lists for domain " + domain + ":")
-	for _, taskList := range response.GetTaskListNames() {
-		fmt.Println(taskList)
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetBorder(true)
+	table.SetColumnSeparator("|")
+	table.SetHeader([]string{"Task List Name", "Type", "Poller Number"})
+	table.SetHeaderLine(true)
+	table.SetHeaderColor(tableHeaderBlue, tableHeaderBlue, tableHeaderBlue)
+	for name, taskList := range response.GetDecisionTaskListMap() {
+		table := tablewriter.NewWriter(os.Stdout)
+		table.SetBorder(true)
+		table.SetColumnSeparator("|")
+		table.SetHeader([]string{"Task List Name", "Type", "Poller Number"})
+		table.SetHeaderLine(true)
+		table.SetHeaderColor(tableHeaderBlue, tableHeaderBlue, tableHeaderBlue)
```

Comment:
I may miss something. why do we need to create a new table and set the header, color, border again here?

Reply:
No need. removed those.